### PR TITLE
SVG feComposite improvements follow-up

### LINKED ISSUE #1450: feComposite docs/support is a mess
@PeterConstable commented on [Tue Dec 01 2020](https://github.com/mdn/sprints/issues/3933)

# Request type

- [x] Correction or update

# Details
This pertains to the SVG feComposite element, as documented in MDN at <https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feComposite>.

There is a complex example at the bottom of that page. Supposedly, per [CanIUse](https://caniuse.com/?search=fecomp), this is well supported in Firefox, Chrome and Edge. However, I cannot find a single implementation that reproduces the results shows at the bottom of that MDN page—not even close! Not Firefox; not Chrome; not Edge; not Adobe Illustrator...

So, I can't tell if the MDN documentation is correct, but nothing actually supports this (in spite of CanIUse reports), or if the MDN documentation is wrong.

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
This PR addresses some more minor bits I discovered while reviewing

Signed-off-by: André Jaenisch <andre.jaenisch@posteo.de>

### CODE REVIEW

File: files/en-us/web/svg/element/fecomposite/index.html [hunk 1]
```diff
@@ -19,46 +19,45 @@
     <th>Description</th>
   </tr>
   <tr>
-    <td><p>over <img src="operation_over.png" title="over operator" width="300"></p></td>
+    <td><p>over <img src="operation_over.png" alt="over operator" width="300"></p></td>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/element/fecomposite/index.html:22]: Should we keep a `title` attribute?
As far as I know, it's bad practice to use the same value for `title` and `alt`.
However, the images have a meaning, so a value for `alt` is required.

File: files/en-us/web/svg/element/fecomposite/index.html [hunk 2]
```diff
@@ -19,46 +19,45 @@
...
     </td>
   </tr>
   <tr>
-    <td><p>in <img src="operation_in.png" title="in operator"></p></td>
+    <td><p>in <img src="operation_in.png" alt="in operator"></p></td>
     <td>The parts of the source graphic defined by the <code>in</code> attribute that overlap the destination graphic defined in the <code>in2</code> attribute, replace the destination graphic.</td>
   </tr>
   <tr>
-    <td><p>over <img src="operation_out.png" title="out operator"> </p></td>
+    <td><p>out <img src="operation_out.png" alt="out operator"> </p></td>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/element/fecomposite/index.html:32]: This was likely a copy-paste-error in #2353 :)

File: files/en-us/web/svg/element/fecomposite/index.html [hunk 3]
```diff
@@ -19,46 +19,45 @@
...
   <tr>
-      <td><p>arithmetic <img src="operation_arithmetic.png" title="arithmetic operator"></p></td>
-      <td>
-        <p>The <code>arithmetic</code> operation is useful for combining the output from the {{SVGElement("feDiffuseLighting")}} and {{SVGElement("feSpecularLighting")}} filters with texture data. If the <code>arithmetic</code> operation is chosen, each result pixel is computed using the following formula:</p>
+    <td><p>arithmetic <img src="operation_arithmetic.png" alt="arithmetic operator"></p></td>
+    <td>
+      <p>The <code>arithmetic</code> operation is useful for combining the output from the {{SVGElement("feDiffuseLighting")}} and {{SVGElement("feSpecularLighting")}} filters with texture data. If the <code>arithmetic</code> operation is chosen, each result pixel is computed using the following formula:</p>
 
-        <pre class="brush: plain">result = k1*i1*i2 + k2*i1 + k3*i2 + k4</pre>
+      <pre class="brush: plain">result = k1*i1*i2 + k2*i1 + k3*i2 + k4</pre>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/element/fecomposite/index.html:52]: Shall this become MathML instead?

File: files/en-us/web/svg/element/fecomposite/index.html [hunk 4]
```diff
@@ -70,28 +69,26 @@ <h2 id="Attributes">Attributes</h2>
 <h3 id="Global_attributes">Global attributes</h3>
 
 <ul>
- <li><a href="/en-US/docs/Web/SVG/Attribute#core_attributes">Core attributes</a> »</li>
- <li><a href="/en-US/docs/Web/SVG/Attribute#presentation_attributes">Presentation attributes</a> »</li>
- <li><a href="/en-US/docs/Web/SVG/Attribute#filter_primitive_attributes">Filter primitive attributes</a> »</li>
- <li>{{SVGAttr("class")}}</li>
- <li>{{SVGAttr("style")}}</li>
+  <li><a href="/en-US/docs/Web/SVG/Attribute#core_attributes">Core attributes</a> »</li>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/element/fecomposite/index.html:72]: I'm a bit unsure regarding the trailing » here. Feels like it's quoting?

File: files/en-us/web/svg/element/fecomposite/index.html [hunk 5]
```diff
@@ -70,28 +69,26 @@ <h2 id="Attributes">Attributes</h2>
 <h3 id="Global_attributes">Global attributes</h3>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/element/fecomposite/index.html:69]: This is actually a bumping head, which violates our writing guidelines:
https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Writing_style_guide#Heading_dos_and_donts

File: files/en-us/web/svg/element/fecomposite/index.html [hunk 6]
```diff
@@ -70,28 +69,26 @@ <h2 id="Attributes">Attributes</h2>
...
 
-
 <h2 id="DOM_Interface">DOM Interface</h2>
 
 <p>This element implements the {{domxref("SVGFECompositeElement")}} interface.</p>
 
-
 <h2 id="Example">Example</h2>
 
 <p>This example defines filters for each of the supported operations (<code>over</code>, <code>atop</code>, <code>lighter</code>, etc.), which composite an input <code>SourceGraphic</code> with an image of the MDN logo. The filters are each applied to a circle element, which is then used as the <code>SourceGraphic</code>. The relevant code for just one filter (<code>lighter</code>) is shown below:</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/element/fecomposite/index.html:94]: Following this paragraph is an indented `<pre>`.
Shall I unindent it?

**Comment by Ryuno-Ki** [comment]
This is ready for review. I commented bits where I felt a bit unsure on what to do.

File: files/en-us/web/svg/element/fecomposite/index.html (hunk 6 above)
Comment (avivmu) [reply files/en-us/web/svg/element/fecomposite/index.html:94]: Seems `<pre>` shouldn't be indented

File: files/en-us/web/svg/element/fecomposite/index.html (hunk 4 above)
Comment (avivmu) [reply files/en-us/web/svg/element/fecomposite/index.html:72]: Quoting what? Seems redundant to me

File: files/en-us/web/svg/element/fecomposite/index.html (hunk 5 above)
Comment (avivmu) [reply files/en-us/web/svg/element/fecomposite/index.html:69]: Well if you are not sure how to fix it you can open an issue for it

File: files/en-us/web/svg/element/fecomposite/index.html (hunk 4 above)
Comment (Ryuno-Ki) [reply files/en-us/web/svg/element/fecomposite/index.html:72]: Chevrons are an alternative to double quotes. I couldn't see anything within them.

File: files/en-us/web/svg/element/fecomposite/index.html (hunk 5 above)
Comment (Ryuno-Ki) [reply files/en-us/web/svg/element/fecomposite/index.html:69]: Sounds good. I noticed them in several places by now.

**Comment by Ryuno-Ki** [comment]
Thanks for the feedback, @avivmu

File: files/en-us/web/svg/element/fecomposite/index.html (hunk 5 above)
Comment (Ryuno-Ki) [reply files/en-us/web/svg/element/fecomposite/index.html:67]: Filled https://github.com/mdn/content/issues/2545 to talk about it.

**Comment by hamishwillee** [comment]
Thanks @Ryuno-Ki . I'm around typically Mon, Tues, Fri AEST, so if you see me ignoring feedback comments, that's why!

**Comment by Ryuno-Ki** [comment]
> Thanks @Ryuno-Ki . I'm around typically Mon, Tues, Fri AEST, so if you see me ignoring feedback comments, that's why!

No hurry. To me, it's entirely spare time. So I don't have a fixed schedule at all :-)

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**